src/libsync/theme.cpp

```diff
@@ -1036,6 +1036,15 @@ bool Theme::darkMode() const
     return isDarkFromStyle();
 }
 
+bool Theme::displayLegacyImportDialog() const
+{
+#if defined APPLICATION_DISPLAY_LEGACY_IMPORT_DIALOG
```

Comment:
```suggestion
#if defined APPLICATION_DISPLAY_LEGACY_IMPORT_DIALOG && APPLICATION_DISPLAY_LEGACY_IMPORT_DIALOG
```
when using `#cmakedefine01`, the macro is always defined but gets a value `0` or `1`